tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -39,35 +39,39 @@ async def test_function_reentrant_clock(dut):
 async def test_timer_with_units(dut):
     time_fs = get_sim_time(units='fs')
 
-    # Await for one simulation time step
+    # Await for one simulator time step
     await Timer(1)  # NOTE: explicitly no units argument here!
     time_step = get_sim_time(units='fs') - time_fs
 
+    time_fs = get_sim_time(units='fs')
+    await Timer(1, units='step')
+    assert time_step == get_sim_time(units='fs') - time_fs, "Expected units='step' == units=None"
+
     try:
         # Await for 2.5 timesteps, should throw exception
         await Timer(2.5*time_step, units='fs')
-        raise TestFailure("Timers should throw exception if time cannot be achieved with simulator resolution")
+        assert False, "Timers should throw exception if time cannot be achieved with simulator resolution"
```

Comment:
If you're changing this, may as well use `assert_raises`

Reply:
Done.